release-calendar/src/client/stylesheets/square.scss

```diff
@@ -0,0 +1,41 @@
+.square-lts {
```

Comment:
There's a lot of repetition here. You can do something like `.lts > .square` to specify the background color, or leverage SCSS mixins using the channel as a param - https://sass-lang.com/documentation/at-rules/mixin#arguments

Reply:
oh, cool. I didnt realize mixins could take in arguments. Thank you!